etc/ci/upload_nightly.sh

```diff
@@ -16,16 +16,60 @@ usage() {
 
 
 upload() {
-    local nightly_filename nightly_timestamp
-    nightly_timestamp="$(date -u +"%Y-%m-%dT%H-%M-%SZ")"
-    nightly_filename="${nightly_timestamp}-$(basename "${2}")"
-    local -r nightly_upload_dir="s3://servo-builds/nightly/${1}"
+    local -r platform="${1}"
+    local -r package="${2}"
+    local -r extension="${3}"
+
+    local -r nightly_timestamp="$(date -u +"%Y-%m-%dT%H-%M-%SZ")"
+    local -r nightly_filename="${nightly_timestamp}-$(basename "${package}")"
+    local -r nightly_upload_dir="s3://servo-builds/nightly/${platform}"
     local -r package_upload_path="${nightly_upload_dir}/${nightly_filename}"
     s3cmd --mime-type="application/octet-stream" \
-          put "${2}" "${package_upload_path}"
-    s3cmd cp "${package_upload_path}" "${nightly_upload_dir}/servo-latest.${3}"
+      put "${package}" "${package_upload_path}"
+    s3cmd cp "${package_upload_path}" \
+      "${nightly_upload_dir}/servo-latest.${extension}"
+
+    if [[ "${platform}" == "macbrew" ]]
+    then
+      update_brew ${nightly_filename} ${package} ${nightly_timestamp}
+    fi
 }
 
+update_brew() {
+  echo "Updating brew formula"
+
+  local -r nightly_filename="${1}"
+  local -r package="${2}"
+  local -r nightly_timestamp="${3}"
+
+  local package_url sha version script_dir tmp_dir
+
+  package_url="https://download.servo.org/nightly/macbrew/${nightly_filename}"
+  sha=$(shasum -a 256 ${package} | sed -e 's/ .*//')
```

Comment:
Make sure to double-quote variables and command substitutions per the [style guide](https://github.com/servo/servo/blob/master/docs/STYLE_GUIDE.md).